cli/src/cardCommands.js

```diff
@@ -188,12 +191,50 @@ const cardCommands = {
         }
     },
 
+    prepareCard(card, update) {
+        const now = new Date().getTime();
+        let jsonCard = JSON.parse(card);
+        if (update) {
+            const jsonUpdate = JSON.parse(update);
+            this.patchCard(jsonCard, jsonUpdate);
+        }
+
+        if (jsonCard.startDate != undefined) jsonCard.startDate = now + jsonCard.startDate;
+        if (jsonCard.endDate != undefined) jsonCard.endDate = now + jsonCard.endDate;
+        if (jsonCard.lttd != undefined) jsonCard.lttd = now + jsonCard.lttd;
+        if (jsonCard.expirationDate != undefined) jsonCard.expirationDate = now + jsonCard.expirationDate;
+
+        jsonCard.timeSpans?.forEach(timespan => {
+            if (timespan.start != undefined) timespan.start = now + timespan.start;
+            if (timespan.end != undefined) timespan.end = now + timespan.end;
+        });
+        return JSON.stringify(jsonCard);
+    },
+
+    patchCard(card, update) {
+        try {
+            for (const [key, value] of Object.entries(update)) {
+                if (
+                    Object.hasOwn(card, key) &&
+                    value != null
+                ) {
+                    card[key] = value;
+                }
+            }
+        } catch (error) {
+            this.logger.error(error);
+        }
+
+        return card;
+    },
+
     async printHelp() {
         console.log(`Usage: opfab card <command> <cardId|cardFileName>
 
 Command list :
 
-    send               send a card : opfab card send <cardFileName>
+    send               send a card : opfab card send <cardFileName> <cardCustomization>
+                       (date values in card must be set as relative to current time)
     delete             delete a card by id : opfab card delete <cardId>
```

Comment:
- Add [\<cardCustomization\>] at the end of usage line 
- Add [\<cardCustomization>\] instead of \<cardCustomization\> as it is an optional  parameter
- Add an explanation for card Customization